src/main/java/org/obsidiantoaster/generator/rest/ObsidianResource.java

```diff
@@ -293,6 +299,42 @@ public Response downloadZip(JsonObject content,
       }
    }
 
+   @POST
+   @javax.ws.rs.Path("/commands/{commandName}/catapult")
+   @Consumes(MediaType.APPLICATION_JSON)
+   public Response uploadZip(JsonObject content,
+                               @PathParam("commandName") @DefaultValue(DEFAULT_COMMAND_NAME) String commandName,
+                               @Context HttpHeaders headers)
+         throws Exception
+   {
+      Response response = downloadZip(content, commandName, headers);
+      if (response.getStatus() != 200) {
+         return response;
+      }
+
+      String fileName = findArtifactId(content);
+      byte[] zipContents = (byte[]) response.getEntity();
+      Client client = ClientBuilder.newBuilder().build();
+      WebTarget target = client.target("http://catapult.local/api/catapult/upload");
```

Comment:
Def needs to be externally configurable; ENV VAR is how I've done this in Catapult